server.js

```diff
@@ -24,9 +27,130 @@ const app = express();
 app.use(cors());
 app.use(express.json());
 
+const Song = mongoose.model('Song', {
+  id: Number,
+  trackName: String,
+  artistName: String,
+  genre: String,
+  bpm: Number,
+  energy: Number,
+  danceability: Number,
+  loudness: Number,
+  liveness: Number,
+  valence: Number,
+  length: Number,
+  acousticness: Number,
+  speechiness: Number,
+  popularity: Number
+});
+
+const Movie = mongoose.model('Movie', {
+  show_id: Number,
+  title: String,
+  director: String,
+  cast: String,
+  country: String,
+  date_added: String,
+  release_year: Number,
+  rating: String,
+  duration: String,
+  listed_in: String,
+  description: String,
+  type: String
+});
+
+if (process.env.RESET_DB) {
+  const resetDataBase = async () => {
+    await Song.deleteMany();
+    topMusicData.forEach((singleSong) => {
+      const newSong = new Song(singleSong);
+      newSong.save();
+    });
+    netflixData.forEach((singleMovie) => {
+      const newMovie = new Movie(singleMovie);
+      newMovie.save();
+    });
+  };
+
+  resetDataBase();
+  console.log('database dropped and recreated');
+}
+
 // Start defining your routes here
-app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+app.get('/', (req, res) => {
+  res.json(listEndpoints(app));
+});
+
+app.get('/songs', async (req, res) => {
+  const allTheSongs = await Song.find({});
+  res.status(200).json({
+    success: true,
+    body: allTheSongs
+  });
+});
+
+app.get('/songs/id/:id', async (req, res) => {
+  try {
+    const singleSong = await Song.findById(req.params.id);
+    console.log(singleSong);
+
+    if (singleSong) {
+      res.status(200).json({
+        success: true,
+        body: singleSong
+      });
+    } else {
+      res.status(404).json({
+        success: false,
+        body: {
+          message: 'Song was not found'
+        }
+      });
+    }
+  } catch (error) {
+    res.status(400).json({
+      success: false,
+      body: {
+        message: 'Invalid id'
+      }
```

Comment:
Great job with the status and error-messages. I've tested them and they work like they should. 👏